docs/specifications/tests/Consistency-TP/consistency05.md

```diff
@@ -1,49 +1,205 @@
-## CONSISTENCY05: Consistency between glue and authoritative data
+# CONSISTENCY05: Consistency between glue and authoritative data
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY05:** Consistency between glue and authoritative data
+**CONSISTENCY05**
 
-### Objective
+## Objective
 
 For name servers that have IP addresses listed as glue, the IP addresses must
-match the authoritative A and AAAA records for that host. This is an IANA
-requirement (https://www.iana.org/help/nameserver-requirements).
+match the authoritative A and AAAA records for that host. This is an IANA 
+[name server requirement].
+
+The objective of this test is to verify that the glue address records 
+in the delegation are consistent with authoritative data.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Ordered description of steps to be taken to execute the test case
+1. Obtain the set of name server names from the NS records in the 
+   delegation of *Child Zone* using [Method2] and any glue IP addresses
+   from the same delegation using [Method4].
+
+   1. Extract the [in-bailiwick] name server names and create the set
+      "Delegation Strict Glue", where each name server name 
+      is matched with its IP address or addresses, if available. The 
+      set may be empty.
+
+   2. Extract the [out-of-bailiwick] name server names and create the 
+      set "Delegation Extended Glue", where each name server name 
+      is matched with its IP address or addresses, if available. The 
+      set may be empty.
+
+2. Obtain the set of name server names for the *Child Zone* using
+   [Method2] and [Method3] and extract the [in-bailiwick] name 
+   server names, "IB NS Name Set".
+
+3. Create an empty set of name server name with associated IP address
+   or addresses, "Address Records From Child".
+
+4. Obtain the set of name server IP addresses for *Child Zone* 
+   using [Method4] and [Method5], "NS IP", if *IB NS Name Set* is 
+   non-empty.
```

Comment:
This does not address my concern. May I suggest this instead?
> Obtain the set of name server IP addresses, "NS IP", for *Child Zone* using [Method4] and [Method5]. (This step can be skipped if *IB NS Name Set* is empty).

Reply:
Updated.

Reply:
I want to execute the step to get an empty set because lowering the algorithm complexity in the specification is much more important than saving CPU cycles in the specification.

Reply:
When reading the specification it makes more sense to ignore a useless step. I do not agree that it lowers the complexity. It is just the same if-statement as in the next step.

@mattias-p, if this is the only objection you have, please approve.